lib/matplotlib/axes/_axes.py

```diff
@@ -753,9 +753,10 @@ def text(self, x, y, s, fontdict=None, **kwargs):
         self._add_text(t)
         return t
 
+    @cbook._rename_parameter("3.2", "s", "text")
```

Comment:
Next release is 3.3; 3.2 is out already.

Reply:
I guess you noted that earlier, but unless something is broken, I'd rather we leave this deprecation for the next version.